hack/lib/update_olm_manifest.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+filename="olm-bindata-"$1.go
+echo $filename
+
+function get_olm_manifests() {
```

Comment:
This function should default to getting the latest release of OLM. If a broken OLM release occurs, we can simply not update bindata or pin to a later non-broken release.

Reply:
@joelanford had suggested that instead of defaulting to the latest release always, it would be better if we could control the version of OLM which is going to get packaged with SDK. It would be a part of release process (an additional step) to run `make olm-manifests` to update  the latest release we want to include after checking that it isn't broken.